From a1bdc32281eff914504e5debb91ae2d3afa42b03 Mon Sep 17 00:00:00 2001
From: "Syed Mohamad Fauzi, Syed Johan Arif"
 <syed.johan.arif.syed.mohamad.fauzi@intel.com>
Date: Sat, 15 Sep 2018 10:41:10 +0800
Subject: [PATCH 4/4] igb: Fine-tune driver offset for time synchronization

This patch is to tune the i210 latency for Intel LeafHill/APL platform.

Signed-off-by: Syed Mohamad Fauzi, Syed Johan Arif <syed.johan.arif.syed.mohamad.fauzi@intel.com>
---
 drivers/net/ethernet/intel/igb/igb.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/intel/igb/igb.h b/drivers/net/ethernet/intel/igb/igb.h
index 1e849a0..621421a 100644
--- a/drivers/net/ethernet/intel/igb/igb.h
+++ b/drivers/net/ethernet/intel/igb/igb.h
@@ -97,10 +97,10 @@
 /* Transmit and receive latency (for PTP timestamps) */
 #define IGB_I210_TX_LATENCY_10		9542
 #define IGB_I210_TX_LATENCY_100		1024
-#define IGB_I210_TX_LATENCY_1000	178
+#define IGB_I210_TX_LATENCY_1000	0
 #define IGB_I210_RX_LATENCY_10		20662
 #define IGB_I210_RX_LATENCY_100		2213
-#define IGB_I210_RX_LATENCY_1000	448
+#define IGB_I210_RX_LATENCY_1000	453
 
 struct vf_data_storage {
 	unsigned char vf_mac_addresses[ETH_ALEN];
-- 
1.9.1

